.cspell/en-words.txt

```diff
@@ -69,6 +69,7 @@ lifecycles
 lightstep
 Loffay
 Mancuso
+matlab
```

Comment:
@chalin since I can not add a cspell ignore section to the JSON file I added it globally. I hope this is a valid exception.

Reply:
If you can add arbitrary fields to the JSON file, which I think we can, then add it via:

```json
  "spelling": "cSpell:ignore words-to-ignore -"
```